src/dashboard/src/components/auth/AuthState.tsx

```diff
@@ -35,22 +36,14 @@ export const AuthState: React.FC<IAuthState> = ({ showName }) => {
     return (
       <Row className="profile">
         {showName && <div>{session?.user.name}</div>}
-        <button onClick={() => keycloakSessionLogOut()} title="Sign out">
-          <div className="group flex flex-row gap-1 items-center text-white-100 hover:text-gray-500 active:text-gray-900 font-bold">
-            <FaSignOutAlt />
-            Sign out
-          </div>
-        </button>
+          <button className={styles.button} onClick={() => keycloakSessionLogOut()} title="Sign out"><FaSignOutAlt />&nbsp;Sign out</button>
```

Comment:
This button should use our `Button` component.  Replace and remove custom styling here `className={styles.button}`